.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
```

Comment:
Why check daily? Given we reference mutable major version tags, and there have been 3 major releases of these actions in as many years, this seems excessive to me.

Reply:
We can change that to a weekly or monthly basis, but since we're using major versions only, which only release sporadically, there's no real benefit in using longer intervals. It's not like dependabot would notify you about every run individually.

Reply:
Right, I guess the workflow run spam only matters once there's more than one workflow in a repo, which is unlikely with this one.